src/async_fuse/fuse/abi_marker.rs

```diff
@@ -110,16 +110,16 @@ mark_sized_types!(@kernel size_check: check_common,
     FuseInHeader,
     FuseOutHeader,
     FuseDirEnt,
-
-    // TODO: remove this after implement corresponding FUSE operations
-    // ---
-    FuseIoCtlIn,
-    FusePollIn,
-    FuseBatchForgetIn,
-    FuseForgetOne,
-    // ---
 );
 
+// TODO: remove this after implement corresponding FUSE operations
```

Comment:
Why are they moved out?

Reply:
Because they are seperated into different ABI versions: `FuseIoCtlIn` and `FusePollIn` for 7.11, `FuseBatchForgetIn` and `FuseForgetOne` for 7.16. Now these structs are marked with correct ABI flags, and they need to be put in the correct macro call, so the TODO comment is moved out.